pkgs/applications/virtualization/virtualbox/qt-env-vars.patch

```diff
@@ -0,0 +1,15 @@
+--- a/src/VBox/HostDrivers/Support/SUPR3HardenedMain.cpp
++++ b/src/VBox/HostDrivers/Support/SUPR3HardenedMain.cpp
+@@ -2131,6 +2131,12 @@ static void supR3HardenedMainPurgeEnvironment(char **envp)
+         /** @todo Call NT API to do the same. */
+ #endif
+     }
++
++    /*
++     * NixOS hack: Set QT_PLUGIN_PATH to make Qt find plugins. The placeholder
++     * on the right will be replaced with the correct value by the builder.
++     */
++    setenv("QT_PLUGIN_PATH", "<QT_PLUGIN_PATH>", /*overwrite=*/ 1);
```

Comment:
Prepare for this to be a reference point for when anyone else bumps into an issue like this :smile:  
(and suddenly there's 15 more patches like it)